eng/packaging.targets

```diff
@@ -42,4 +54,76 @@
              Targets="Build" />
   </Target>
+    
+  <Target Name="AddRefAssemblies" Condition="'$(IncludeRefInPackage)' == 'true' and '$(TargetsAnyOS)' == 'true' and '@(ResolvedMatchingContract)' != ''"> 
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="@(ResolvedMatchingContract)">
+        <RefAssemblyOutput>true</RefAssemblyOutput>
+        <PackagePath>ref/$([System.String]::new('%(ResolvedMatchingContract.SetTargetFramework)').Substring(16))</PackagePath>
+        <PackagePathWithFilename>ref/$([System.String]::new('%(ResolvedMatchingContract.SetTargetFramework)').Substring(16))/%(ResolvedMatchingContract.Filename).dll</PackagePathWithFilename>
+      </TfmSpecificPackageFile>
+      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)1033\$(TargetName).xml" Condition="Exists('$(XmlDocFileRoot)1033\$(TargetName).xml')">
+        <RefAssemblyOutput>true</RefAssemblyOutput>
+        <PackagePath>ref/$([System.String]::new('%(ResolvedMatchingContract.SetTargetFramework)').Substring(16))</PackagePath>
+        <PackagePathWithFilename>ref/$([System.String]::new('%(ResolvedMatchingContract.SetTargetFramework)').Substring(16))/%(ResolvedMatchingContract.Filename).xml</PackagePathWithFilename>
+      </TfmSpecificPackageFile>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="AddRuntimeSpecificAssemblies"
+          DependsOnTargets="BuiltProjectOutputGroup;SatelliteDllsProjectOutputGroup;_AddPriFileToPackBuildOutput;$(TargetsForTfmSpecificBuildOutput)"
+          Condition="'$(TargetsAnyOS)' != 'true' and '$(ExcludeFromPackage)' == ''">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="@(SatelliteDllsProjectOutputGroupOutput);
+                            @(BuiltProjectOutputGroupOutput);
+                            @(_PathToPriFile)">
+        <PackagePath>runtimes/$(_runtimeOS)/lib/$(TargetFrameworkWithoutSuffix)</PackagePath>
+      </TfmSpecificPackageFile>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="AddRefDebugSymbolsWithTfm"
+          Condition="'$(IncludeSymbols)' == 'true' and '$(IncludeRefInPackage)' == 'true' and '$(TargetsAnyOS)' == 'true'">
+    <PropertyGroup>
+      <RefSymbolPath>$(BaseOutputPath)ref\$(TargetFrameworkWithoutSuffix)-$(Configuration)\$(TargetName).pdb</RefSymbolPath>
+    </PropertyGroup>
+
+    <ItemGroup Condition="Exists('$(RefSymbolPath)')">
+      <TfmSpecificDebugSymbolsFile Include="$(RefSymbolPath)">
+        <TargetPath>/ref/$(TargetFrameworkWithoutSuffix)/$(TargetName).pdb</TargetPath>
+        <TargetFramework>$(TargetFrameworkWithoutSuffix)</TargetFramework>
+      </TfmSpecificDebugSymbolsFile>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="AddRuntimeDebugSymbolsWithTfm"
+          Condition="'$(IncludeSymbols)' == 'true' and '$(TargetsAnyOS)' != 'true' and '$(ExcludeFromPackage)' == ''">
+    <PropertyGroup>
+      <RuntimeSymbolPath>$(BaseOutputPath)$(TargetFrameworkWithoutSuffix)-$(TargetFrameworkSuffix)-$(Configuration)\$(TargetName).pdb</RuntimeSymbolPath>
```

Comment:
If this runs as part of the inner build, could you use `$(TargetPath)` instead and change the extension to .pdb?

Reply:
Hmm why are we adding the symbol from the rid less configuration instead of the rid specific symbol?

ExcludeFromPackage should check not being true, not empty: `'$(ExcludeFromPackage)' != 'true'"`